addon/webextension/background/auth.js

```diff
@@ -199,17 +199,12 @@ this.auth = (function () {
 
   communication.register("getAuthInfo", (sender, ownershipCheck) => {
     let info = registrationInfo;
-    let done = Promise.resolve();
     if (info.registered) {
-      done = login({ownershipCheck}).then((result) => {
-        if (result && result.isOwner) {
```

Comment:
this line ended up being a one-way switch, once it was `isOwner` was true it could never be set back to false.